fitnesse/resources/bootstrap-plus/templates/symlinks.vm

```diff
@@ -0,0 +1,33 @@
+<form method="get" action="" name="symbolics" id="symbolics">
+    <h2>Symbolic Links manager</h2>
```

Comment:
Mierenneukdingetje... is het een Symbolic Link Manager? of moet 'ie eigenlijk 'Symbolic Link Overview' heten?